### ISSUE TITLE
scm tool: delete requires admin access

### ISSUE THREAD

**Issue by meling** [autograde/aguis#46]
Trying to delete repos:
```
scm delete repository --all --namespace autograder-test
Are you sure you want to delete all repositories in autograder-test? (y/N): y
DELETE https://api.github.com/repos/autograder-test/course-info: 403 Must have admin rights to Repository. []
```

Does not work with simple access token obtained from the DB remote identities. (Not sure what is necessary to fix this. @s111 I can fix, if you give me a hint.)

**Comment by s111** [comment]
To delete a repository you need the `delete_repo` scope from GitHub. You need to add the `delete_repo` scope to the `github-teacher` provider. After logging in on the frontend, navigate to `/auth/github-teacher` to request additional rights (remember to grant the orgs you want to modify), the scm tool should then work.

**Comment by meling** [comment]
Doesn't seem very convenient to have to navigate manually. I guess, the frontend should recognize that I'm teacher when logging in, at least after having been assigned a teacher role, and from then it should be automatic, right?

**Comment by s111** [comment]
Yes, when you get promoted to teacher, autograder will force you to navigate to `github-teacher` and authenticate. This is however, not implemented yet, so you'll have to do it manually.

**Comment by meling** [comment]
I tried to hit `/auth/github-teacher` before running the scm command, still same issue. Do I also need to add the delete_repo scope to github-teacher myself? I thought that was the idea of the endpoint?

**Comment by s111** [comment]
The `github-teacher` endpoint currently only requests the additional scope `repos`, just add `delete_repo` to that list and it should work.

**Comment by meling** [comment]
Thanks now it worked.
